generators/docker-compose/index.js

```diff
@@ -78,37 +78,39 @@ module.exports = yeoman.Base.extend({
                 type: 'input',
                 name: 'directoryPath',
                 message: 'Enter the root directory where your gateway(s) and microservices are located',
-                default: '../',
-                validate: function (input) {
```

Comment:
One disadvantage of moving validate from here is that user will not get a chance to correct his error, inquirer will stay on the question for user to rnter correct path normally when there is a validation error